packages/params/src/presets/minimal/altair.ts

```diff
@@ -14,7 +14,7 @@ export const altairJson = {
   DOMAIN_CONTRIBUTION_AND_PROOF: "0x09000000",
 
   ALTAIR_FORK_VERSION: "0x01000001",
-  ALTAIR_FORK_SLOT: "0xffffffffffffffff",
```

Comment:
Why do we change from slot to epoch? Is this how it's defined in the spec?

Reply:
yeah it's changed 8 days ago https://github.com/ethereum/eth2.0-specs/blob/dev/configs/mainnet/altair.yaml